pkg/ocicni/ocicni.go

```diff
@@ -628,7 +628,7 @@ func (plugin *cniNetworkPlugin) getCachedNetworkInfo(containerID string) ([]NetA
 		}{}
 
 		if err := json.Unmarshal(bytes, &cachedInfo); err != nil {
-			logrus.Warningf("failed to unmarshal CNI cache file %s: %v", cacheFile, err)
+			logrus.Errorf("failed to unmarshal CNI cache file %s: %v", cacheFile, err)
```

Comment:
Same question here about the change from "Warning" to "Error".

Reply:
fixed!

Reply:
And same here.